js/widgets/forms/clearButton.js

```diff
@@ -24,55 +27,75 @@ define( [
 				this.options.clearBtn = true;
 			}
 
-			if ( !!this.options.clearBtn && this.inputNeedsWrap ) {
-				this._addClearBtn();
+			// We do nothing on startup if the options is off or if this is not a wrapped input
+			if ( !this.options.clearBtn || !this.inputNeedsWrap ) {
```

Comment:
We also don't wrap range inputs, so we want to avoid adding clear buttons there.